packages/core/src/lib/backend.ts

```diff
@@ -0,0 +1,39 @@
+import { hashToField } from './hashing'
+import type { ISuccessResult } from '../types'
+import { isNode, isDeno } from 'browser-or-node'
+
+export interface IVerifyResponse {
+	success: boolean
+	code?: string
+	detail?: string
+	attribute?: string | null
+}
+
+export async function verifyCloudProof(
+	proof: ISuccessResult,
+	app_id: `app_${string}`,
+	action: string,
+	signal?: string
+): Promise<IVerifyResponse> {
+	if (!isNode && !isDeno) {
+		throw new Error('verifyCloudProof can only be used in the backend.')
+	}
```

Comment:
Can we just do `isBrowser` instead (from the same package)? seems more resilient